admin/spec/service/pin-generation.service.spec.js

```diff
@@ -190,6 +190,10 @@ describe('pin-generation.service', () => {
         const result = pinGenerationService.generateSchoolPassword(school)
         expect(result.pinExpiresAt).toBeDefined()
         expect(result.schoolPin.length).toBe(8)
+        const schoolPin = result.schoolPin
+        expect(/[a-z]{3}/.test(schoolPin.slice(0, 3))).toBe(true)
```

Comment:
A single regex would cover all 3 cases: /[a-z][3}-[2-9]{2}-[a-z]{3}/

Note the numbers 0 and 1 should not be in the output.